includes/http-api/shared-language-mapping.md

```diff
@@ -0,0 +1,1396 @@
+
+If the `language` field in the request contains a tag, the tag will be changed to its human-friendly language name. For example, if a request contains `"language": "en-US"`, it will be changed to `"language": "English"` before it is stored.
+
+The following table shows the list of mappings. Language tags are case-insensitive.
+
+??? mapping "Language Mappings"
```

Comment:
Is mapping a valid admonishment? I haven't seen that before.

Reply:
I have no idea what that word is and what values are valid.  What other word do you suggest?

Reply:
Here is a list of valid values - https://discourse.amplitude.com/t/approve-a-device-mapping/10521